client/my-sites/backup/style.scss

```diff
@@ -67,8 +67,8 @@
 .backup__subheader {
 	margin: 20px 16px 8px;
 
-	font-family: 'Recoleta', Georgia, 'Times New Roman', Times, serif;
-	font-size: rem( 21px );
+	font-family: $brand-serif;
```

Comment:
If you `@extend .wp-brand-font` here, you'll take advantage of the language selectors, which means folks who don't have full support for Recoleta won't see "ransom note" style characters.

See:

https://github.com/Automattic/wp-calypso/blob/00bb0f424a8816da7f11e3f7a37a3a065156e683/packages/typography/sass/fonts.scss#L17-L26

Reply:
Amazing tip, wasn't aware of this piece of code! Thanks, @sixhours.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/43685/commits/a31d53a17873fa2dc0e84acdb387e9e00c9ae747

Reply:
Does that work by extending tho? 🤔 

I'd imagine you have to add `wp-brand-font` class to the component?

Reply:
@simison FWIW, it seems to work: 

![image](https://user-images.githubusercontent.com/390760/85852770-2ec15680-b7a9-11ea-8231-7bd94b70c899.png)

Reply:
Excellent! 🎉